Tests/test_MMCIF.py

```diff
@@ -203,6 +203,52 @@ def test_filehandle(self):
         structure = parser.get_structure("example", open("PDB/1A8O.cif"))
         self.assertEqual(len(structure), 1)
 
+    def test_point_mutations(self):
+        """Test if the parser can parse point mutations correctly."""
+
+        def _run_parser(parser):
+            structure = parser.get_structure("example", "PDB/3JQH.cif")
+
+            # Residue 1 and 15 should be disordered.
+            res_1 = structure[0]["A"][1]
+            res_15 = structure[0]["A"][15]
+
+            # Cursory check -- this would be true even if the residue just
+            # contained some disordered atoms.
+            self.assertTrue(res_1.is_disordered(), "Residue 1 is disordered")
+            self.assertTrue(res_15.is_disordered(), "Residue 15 is disordered")
+
+            # Check a non-mutated residue just to be sure we didn't break the
+            # parser and cause everyhing to be disordered.
+            self.assertFalse(
+                structure[0]["A"][13].is_disordered(),
+                "Residue 13 is not disordered")
+
+            # Check that the residue types were parsed correctly.
+            self.assertSetEqual(
+                set(res_1.disordered_get_id_list()),
+                {"PRO", "SER"},
```

Comment:
This is fun to see - it might be the first set literal in Biopython as we've only just dropped support for Python 2.6 :)